docs/pages/install/macos.mdx

```diff
@@ -53,12 +55,12 @@ This document explains how to install Pelican on macOS.
     Built By: goreleaser
     ```
 
-    * Add Pelican binary to your `PATH` permanently
+  * Add Pelican binary to your `PATH` permanently
 
     ```bash
     $ cd pelican_Darwin_arm64 # Go to the binary folder
-    $ echo 'export PATH=$PWD:$PATH' >> ~/.bashrc # Add the current folder to your .bashrc file
-    $ source ~/.bashrc # Apply the change
+    $ echo 'export PATH=$PWD:$PATH' >> ~/.zshrc # Add the current folder to your .zshrc file
```

Comment:
See note in `linux_binary.mdx` for more information.

```suggestion
    $ echo "export PATH=$PWD:\$PATH" >> ~/.zshrc # Add the current folder to your .zshrc file
```

Might want someone to test that this works as expected.